packages/config/config/devices/0x0346/motion_sensor_gen2.json

```
@@ -0,0 +1,195 @@
+// Ring - / 4SP1SZ-0EU0
+// Motion Sensor Gen 2
+{
+	"manufacturer": "Ring",
+	"manufacturerId": "0x0346",
+	"label": "- / 4SP1SZ-0EU0",
+	"description": "Motion Sensor Gen2",
+	"devices": [
+		{
+			"productType": "0x0301",
+			"productId": "0x0301"
+		},
+		{
+			"productType": "0x0301",
+			"productId": "0x0401"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Heartbeat Interval",
+			"valueSize": 1,
+			"unit": "minutes",
+			"minValue": 1,
+			"maxValue": 70,
+			"defaultValue": 70,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"2": {
+			"label": "Application Retries",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 5,
+			"defaultValue": 5,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"3": {
+			"label": "Application Level Retries",
+			"valueSize": 1,
+			"unit": "seconds",
+			"minValue": 1,
+			"maxValue": 96,
+			"defaultValue": 5,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"4": {
+			"label": "Led Indicator",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Don't show green",
+					"value": 0
+				},
+				{
+					"label": "Show green after Fault",
+					"value": 1
+				},
+				{
+					"label": "Show green after Intrusion and Intrusion clear",
+					"value": 2
+				}
+			]
```

Are there more colors than green? If there's only green, I would do something like
```suggestion
			"options": [
				{
					"label": "Disable",
					"value": 0
				},
				{
					"label": "Light up after fault",
					"value": 1
				},
				{
					"label": "Light up after intrusion and intrusion clear",
					"value": 2
				}
			]
```